index.js

```diff
@@ -180,7 +180,8 @@ module.exports = (leftLib, rightLib, options, specs) => {
       style: colors ? {} : {border: [], head: []},
     });
 
-    const keys = (Object.keys (specs)).filter (micromatch.matcher (match));
+    const isMatch = picomatch (match);
+    const keys = (Object.keys (specs)).filter (key => isMatch (key));
```

Comment:
I used `key => isMatch (key)` to avoid problems with `Array#filter` and optional arguments.